src/Network/Network/Models/PSNetworkInterfaceIpConfiguration.cs

```diff
@@ -48,7 +48,7 @@ public class PSNetworkInterfaceIPConfiguration : PSIPConfiguration
 
         [JsonProperty(Order = 2)]
         [Ps1Xml(Target = ViewControl.Table)]
-        public PSFrontendIPConfiguration GatewayLoadBalancer { get; set; }
+        public PSResourceId GatewayLoadBalancer { get; set; }
```

Comment:
I'm hesitate to approve this because type change will cause breaking changes in many cmdlets even thought it's a bug fix...Is it acceptable if we introduce this in May?

Reply:
this is not a breaking change because it never worked. it was basically a typo. I think it's better to push in now because we have impacted customers

Reply:
I agree